src/Microsoft.TestPlatform.Common/Resources/xlf/Resources.cs.xlf

```diff
@@ -269,6 +269,11 @@
         <target state="translated">Operace se ruší na základě žádosti.</target>
         <note />
       </trans-unit>
+      <trans-unit id="CannotFetchByFriendlyName">
+        <source>Cannot Fetch Uri with {0}.</source>
```

Comment:
Unable to find a datacollector with friendly name '[0}'

Reply:
changed as mentioned